perf/lua/1mops_write.lua

```diff
@@ -0,0 +1,323 @@
+local err, popen = pcall(require,'popen')
```

Comment:
Currently it is not possible to run a script without specifying interpreter:
```
$ ./perf/lua/1mops_write.lua
-bash: ./perf/lua/1mops_write.lua: Permission denied    
```

I would add a shebang:

```diff
--- a/perf/lua/1mops_write.lua
+++ b/perf/lua/1mops_write.lua
@@ -1,3 +1,5 @@
+#!/bin/env tarantool
+
 local err, popen = pcall(require,'popen')
 if err ~= true then
     print(err)
```

and give exec permissions:

```
$ chmod +x perf/lua/1mops_write.lua 
$ ./perf/lua/1mops_write.lua 
# making 10000000 REPLACE operations,
...
```